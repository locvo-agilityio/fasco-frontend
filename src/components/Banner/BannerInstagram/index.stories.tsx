import type { Meta, StoryObj } from '@storybook/react';

// Components
import BannerInstagram from '.';

const meta = {
  title: 'Components/BannerInstagram',
  component: BannerInstagram,
  parameters: {
    layout: 'centered',
  },
  render: () => (
    <div className="w-[1920px]">
      <BannerInstagram />
    </div>
  ),
  tags: ['autodocs'],
} satisfies Meta<typeof BannerInstagram>;

export default meta;
type Story = StoryObj<typeof meta>;

export const Default: Story = {};
