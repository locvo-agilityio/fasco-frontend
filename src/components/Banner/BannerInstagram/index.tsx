import Image from 'next/image';

// Images
import InstagramImage from '../../../../public/assets/image-instagram.webp';

const BannerInstagram = () => (
  <div className="bg-gradient-white flex flex-col items-center justify-center w-full">
    <h3 className="text-4xl font-volkhov text-primary-200 text-center mt-16">
      Follow Us On Instagram
    </h3>
    <p className="max-w-[614px] text-center text-gray-500 font-poppins mt-5">
      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Scelerisque duis
      ultrices sollicitudin aliquam sem. Scelerisque duis ultrices sollicitudin
    </p>

    <div className="relative w-full max-w-[1920px] h-[380px] mt-24">
      <Image
        alt="Sign In Image"
        src={InstagramImage}
        quality={100}
        fill
        sizes="100vw"
        style={{ objectFit: 'cover' }}
        priority
      />
    </div>
  </div>
);

export default BannerInstagram;
