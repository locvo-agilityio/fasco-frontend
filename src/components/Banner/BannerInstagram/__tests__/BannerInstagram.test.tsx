import { render } from '@testing-library/react';

// Components
import BannerInstagram from '..';

describe('BannerInstagram', () => {
  it('should render correctly', () => {
    const { container } = render(<BannerInstagram />);

    expect(container).toMatchSnapshot();
  });
});
