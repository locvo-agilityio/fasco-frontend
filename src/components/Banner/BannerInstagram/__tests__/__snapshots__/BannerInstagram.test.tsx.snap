// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BannerInstagram should render correctly 1`] = `
<div>
  <div
    class="bg-gradient-white flex flex-col items-center justify-center w-full"
  >
    <h3
      class="text-4xl font-volkhov text-primary-200 text-center mt-16"
    >
      Follow Us On Instagram
    </h3>
    <p
      class="max-w-[614px] text-center text-gray-500 font-poppins mt-5"
    >
      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Scelerisque duis ultrices sollicitudin aliquam sem. Scelerisque duis ultrices sollicitudin
    </p>
    <div
      class="relative w-full max-w-[1920px] h-[380px] mt-24"
    >
      <img
        alt="Sign In Image"
        data-nimg="fill"
        decoding="async"
        sizes="100vw"
        src="/_next/image?url=%2Fimg.jpg&w=3840&q=100"
        srcset="/_next/image?url=%2Fimg.jpg&w=640&q=100 640w, /_next/image?url=%2Fimg.jpg&w=750&q=100 750w, /_next/image?url=%2Fimg.jpg&w=828&q=100 828w, /_next/image?url=%2Fimg.jpg&w=1080&q=100 1080w, /_next/image?url=%2Fimg.jpg&w=1200&q=100 1200w, /_next/image?url=%2Fimg.jpg&w=1920&q=100 1920w, /_next/image?url=%2Fimg.jpg&w=2048&q=100 2048w, /_next/image?url=%2Fimg.jpg&w=3840&q=100 3840w"
        style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; object-fit: cover; color: transparent;"
      />
    </div>
  </div>
</div>
`;
